docs/common/navigation.js

```diff
@@ -78,6 +78,7 @@ const sections = [
       'Routing & Navigation',
       'Authentication',
       'App Icons',
+      'Webhooks',
```

Comment:
this should be under the "Distributing Your App" section right?

also, maybe we should name the page "Build Webhooks"

Reply:
It looked like the webhooks command had been built to eventually support other events since it has the `--event <event-type>` param, so i hesitated to put it under "distributing your app" & calling it "build webhooks" (+ that title to me reads "build webhooks" as in the act of building)